client/state/plans/selectors/get-discounted-raw-price.js

```diff
@@ -15,6 +15,7 @@ export function getDiscountedRawPrice( state, productId, isMonthly = false ) {
 	const plan = getPlan( state, productId );
 	const rawPrice = plan?.raw_price ?? -1;
 	const origCost = plan?.orig_cost ?? -1;
+
 	if ( rawPrice < 0 || origCost < 0 ) {
```

Comment:
This made me realize that this function was missing a failsafe for missing products so I'm adding one here: https://github.com/Automattic/wp-calypso/pull/75140